src/write-archive/index.ts

```diff
@@ -62,16 +62,7 @@ export async function writeTestResult(
     })
   );
 
-  await Object.entries(domSnapshots).map(async ([name, domSnapshot]) => {
-    // XXX_jwir3: We go through our stories here and map any instances that are found in
-    //            the keys of the source map to their respective values.
-    const mappedSnapshot = await mapSourceEntries(domSnapshot, sourceMap);
-
-    await outputFile(
-      join(archiveDir, `${sanitize(title)}-${sanitize(name)}.snapshot.json`),
-      mappedSnapshot
-    );
-  });
+  await writeSnapshotFiles(domSnapshots, archiveDir, title, sourceMap);
```

Comment:
Do you still need the changes in this file?

Reply:
@tevanoff I don't, good catch!